code/game/objects/items/mountable_frames/extinguisher_cabinet.dm

```diff
@@ -0,0 +1,10 @@
+/obj/item/mounted/frame/extinguisher_cabinet
+	name = "extinguisher cabinet frame"
+	desc = "For easy storage of extinguisher cabinets"
```

Comment:
What's the bet on extinguisher being replaced with extinguisher cabinet and this being missed?